mcs/class/corlib/Test/System/TimeZoneInfoTest.cs

```diff
@@ -54,11 +54,11 @@ public static string MapTimeZoneId (string id)
 					return "New Zealand Standard Time";
 				case "Europe/Athens":
 					return "GTB Standard Time";
-				case "US/Eastern":
+				case "America/New_York":
 					return "Eastern Standard Time";
-				case "US/Central":
+				case "America/Chicago":
```

Comment:
There's still a usage of `US/Central` in the `Bug_9664` test.

Reply:
OK, I'm fine if you revert this here and add logic to skip Bug_9664() if US/Central can't be found, e.g. like this:

```diff
			[Test]
			public void Bug_9664 ()
			{
-				TimeZoneInfo tzi = TimeZoneInfo.FindSystemTimeZoneById (MapTimeZoneId ("US/Central"));
+				TimeZoneInfo tzi;
+				try {
+					tzi = TimeZoneInfo.FindSystemTimeZoneById (MapTimeZoneId ("US/Central"));
+				} catch (TimeZoneNotFoundException e) {
+					Assert.Ignore ("Timezone US/Central not found.");
+				}
```